save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FileUploader.kt

```diff
@@ -220,28 +222,33 @@ fun <F : AbstractFileInfo> fileUploader() = FC<UploaderProps<F>> { props ->
     }
 
     val (fileForUploading, setFileForUploading) = useState<File>()
+    @Suppress("TOO_MANY_LINES_IN_LAMBDA")
     val uploadFile = useDeferredRequest {
```

Comment:
looks like you disabled uploading `setup.sh` for all cases, but it should be prohibited only for sandbox mode

Reply:
Are you sure? Probably you meant to fix it in another line but not here.

Reply:
Fixed it in line №260